### DISCUSSION TITLE
Requesting for fully-fledged bookmarking feature

### DISCUSSION THREAD

**Discussion by alwaisy** [dailydotdev/daily#1627] [Feature Requests]
Hi, 

- Since Daily Dev is the start page for most of the engineers.
- Chrome bookmarks are useless. 
- Raindrop requires a separate webpage to open--- in-use

What if Daily Dev introduces Bookmarking features with most of the features from Raindrop? It would be a win for both parties Daily dev and Users.  

## Possible challenges
- Daily Dev is not a bookmarking company. 
- It's Fully focused on micro industry --- better feed for engineers 
- UI obviously was designed with a vision in mind by creative designers. 
- How could bookmarking feature be introduced --- vision breaking / UI breaking / and more

**Comment by idoshamun** [comment]
Hi @alwaisy,

Thanks a lot for the suggestion. We will consider it.
Any specific features you would like to see?
